### PR TITLE
{debugger}[intel/2016a] GDB v7.11

### PR DESCRIPTION
**Pull request by wpoely86**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2413.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0882cdaac91d1e9af998 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2074.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/77eafc7bd3df304b79a5 for a full test report.

File: easybuild/easyconfigs/g/GDB/GDB-7.11-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+source_urls = [GNU_SOURCE]
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('expat', '2.1.0'),
+    ('Python', '2.7.11'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/GDB/GDB-7.11-intel-2016a.eb]: this warrants a versionsuffix (cfr. existing easyconfig for GDB 7.10.1)

File: easybuild/easyconfigs/g/GDB/GDB-7.11-intel-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+name = 'GDB'
+version = '7.11'
+
+homepage = 'http://www.gnu.org/software/gdb/gdb.html'
+description = "The GNU Project Debugger"
+
+sources = [SOURCELOWER_TAR_XZ]
+source_urls = [GNU_SOURCE]
+
+toolchain = {'name': 'intel', 'version': '2016a'}
```
Comment (boegel) [inline easybuild/easyconfigs/g/GDB/GDB-7.11-intel-2016a.eb]: `toolchain` goes above `sources`

File: easybuild/easyconfigs/g/GDB/GDB-7.11-intel-2016a.eb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('expat', '2.1.0'),
+    ('Python', '2.7.11'),
+]
+
+patches = ['GDB-7.9-missing-makeinfo.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/g/GDB/GDB-7.11-intel-2016a.eb]: `patches` goes below `sources`, above `dependencies`

**Comment by boegel** [comment]
@wpoely86 ping on remarks?

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2432.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3889552352b75e9e1abec3e24d5c589d for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2001.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/add263b62f71382ab702a9ae0e5e3f43 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**